### PR TITLE
#3532 Issue with gitignore 

### LINKED ISSUE #3532: Issue with gitignore
**Summary:** 

gitignore includes not to ignore files with path 
-!/.idea/codeStyles/
because of which .idea/codeStyles/Project.xml shows up in changed files

 
**Commons app version:** 

Current master

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by macgills**
**Description (required)**

Fixes #3532 Issue with gitignore 

What changes did you make and why?

synchronise section with default plugin state

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -47,31 +47,9 @@
       <option name="SPACE_BEFORE_SUPERCLASS_COLON" value="false" />
     </Objective-C>
     <Objective-C-extensions>
```
Comment (maskaravivek) [inline .idea/codeStyles/Project.xml]: Is it required to keep the section or can it be removed completely?

**Comment by sivaraam** [comment]
> There are sections for a lot of different languages in this XML file? Can we remove all the irrelevant ones.

I think that would just brings us back to what we're fixing now. Imagine we remove those sections for the sake of discussion. Then, IntelliJ would modify the file to add all those sections that we consider unwanted. Contributors would inadvertently commit those changes and push them. We would ask them to revert those. To avoid such mistakes in future, we would then add those sections again in another PR.

I think we don't need that. Also, I'm just theorizing here. So, correct me if I'm wrong somewhere.

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (macgills) [reply .idea/codeStyles/Project.xml]: You cannot disable a codestyle section 
image

so yeah, required to keep

Comment (maskaravivek) [reply .idea/codeStyles/Project.xml]: In that case we should keep it empty for all other languages too.

**Pull request merged**